themes/ffrn/layouts/partials/head.html

```diff
@@ -1,6 +1,4 @@
-<head>
-    <meta charset="UTF-8">
-    <title> {{- if .IsHome -}} {{ .Site.Title }} {{- else }} {{ .Title }} | {{ .Site.Title }} {{- end -}} </title>
+
     {{ $style := resources.Get "sass/main.scss" | resources.ExecuteAsTemplate "style.main.scss" . | resources.ToCSS | resources.Minify | resources.Fingerprint }}
```

Comment:
Unter Linux gerade den Fehler bekommen:

```
Error: Error building site: TOCSS: failed to transform "style.main.scss"
```

in der README.md fehlt ein Hinweis das die `Hugo Extended`-Version erforderlich ist.